app/danhmucall.jsx

```
@@ -318,103 +401,60 @@ export default function DanhMucAll() {
       }
       params.append('sort', sortParam);
-
-      // Add pagination
       params.append('page', '1');
-      params.append('limit', '50'); // Increased limit for better UX
-
-      // Make API call
+      params.append('limit', '50');
+      // Chỉ truyền filter khi ở tab Bộ lọc
+      if (activeTab === 3) {
+        const catId = filters.category || selectedCategory || categoryId;
```

ESLint

[error] 'selectedCategory' is not defined.

(no-undef)